File: src/SaluteOnline.Client/saluteOnlineApp/app/domain/PlayerEntry.ts
export interface IPlayerEntry {
    nick: string;
    role?: string;
    foul: number;
    bestPlayer: string;
    result: number;
    positionInKillQueue?: number;
    killedAtDay?: boolean;
    killedAtNight?: boolean;
    checkedAtNight?: number;
    rolesAvailable: string[];
    bestPlayersAvailable: string[];
}

export class PlayerEntry implements IPlayerEntry {
    constructor(player: IPlayerEntry = {
        nick: '',
        role: null,
        foul: 0,
        bestPlayer: '',
        result: 0,
        positionInKillQueue: null,
        killedAtDay: null,
        killedAtNight: null,
        checkedAtNight: null,
        rolesAvailable: ['Мирний', 'Чорний', 'Шериф', 'Дон'],
        bestPlayersAvailable: ['Кращий 1', 'Кращий 2', 'Кращий 3']
    }) {
        this.nick = player.nick;
        this.role = player.role;
        this.foul = player.foul;
        this.bestPlayer = player.bestPlayer;
        this.result = player.result;
        this.positionInKillQueue = player.positionInKillQueue;
        this.killedAtDay = player.killedAtDay;
        this.killedAtNight = player.killedAtNight;
        this.checkedAtNight = player.checkedAtNight;
        this.rolesAvailable = player.rolesAvailable;
        this.bestPlayersAvailable = player.bestPlayersAvailable;
    }

    nick: string;
    role: string;
    foul: number;
    bestPlayer: string;
    result: number;
    positionInKillQueue: number;
    killedAtDay: boolean;
    killedAtNight: boolean;
    checkedAtNight: number;
    rolesAvailable: string[];
    bestPlayersAvailable: string[];
}

File: src/SaluteOnline.Domain/Common/PlayerEntry.cs
using SaluteOnline.Domain.Enums;

namespace SaluteOnline.Domain.Common
{
    public class PlayerEntry
    {
        public int Index { get; set; }
        public string Nick { get; set; }
        public Roles Role { get; set; }
        public int? Foul { get; set; }
        public BestPlayers BestPlayer { get; set; }
        public int? Result { get; set; }
        public int? PositionInKillQueue { get; set; }
        public bool KilledAtDay { get; set; }
        public bool KillerAtNight { get; set; }
        public int? CheckedAtNight { get; set; }
        public bool HalfBestWay { get; set; }
        public bool FullBestWay { get; set; }
    }
}